.eslintrc.js

```diff
@@ -52,5 +52,6 @@ module.exports = {
     "migration/",
     "**/*.stories.tsx",
     "**/.eslintrc.js",
+    "tasks/bloom-etl"
```

Comment:
The blended packages here are not the intended use case for yarn workspaces.  Each set of code should be treated as independent if we're following app dev best practices, and forcing tight coupling is detrimental to long-term maintainability.  Magic should be avoided, especially when it isn't documented.  Think about how difficult all the version upgrades have been because all the sites are sharing the same dependencies, many of which are not documented in their respective package.json files.  Or about how slow the build and test times are because they have to pull in dependencies for everything, even on a small local change, and how they are run as a single job rather than multiple in parallel.  I can run the linter and unit tests for this package in parallel, not just to other packages but to themselves, simply by not enforcing a dependency on all of the other code that, frankly, it shouldn't even know about, much less be dependent on.  When I run things locally, it takes single digit seconds to build, lint, and test rather than minutes, which is IMO a big QOL improvement from a dev perspective.

This code in particular is intended to be removed in a relatively short span of time, so keeping it separate makes it much easier to delete everything without any unintended side effects.  It also makes it much easier to pull it into a separate repo if needed at some point, as the only things that are needed are the files within this specific directory and the single workflow file that contains all the jobs needed to validate the code.  I don't think losing that to save a couple of lines in a couple of config files is worth it.